tests/src/actionContainers/SwiftActionContainerTests.scala

```diff
@@ -196,7 +198,29 @@ class SwiftActionContainerTests extends BasicActionRunnerTests with WskActorSyst
         })
     }
 
-        it should "properly use KituraNet and Dispatch" in {
+    it should "support pre-compiled binary in a zip file" in {
```

Comment:
My rationale: 1. Yes the test will break and so we'd have to regenerate the zip file. But this might require tweaking the build steps anyway for the new swift version. 2. I expect we all automate the binary building in a systematic way for native actions in the near future.